src/usb/usb-types.h

```diff
@@ -0,0 +1,87 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <stdint.h>
+#include <sstream>
+#include <string>
+#include <map>
+
+const uint16_t VID_INTEL_CAMERA = 0x8086;
+
+namespace librealsense
+{
+    namespace platform
+    {
+        typedef enum endpoint_direction
+        {
+            USB_ENDPOINT_DIRECTION_WRITE = 0,
+            USB_ENDPOINT_DIRECTION_READ = 0x80
+        } endpoint_direction;
+
+        typedef enum endpoint_type
+        {
+            USB_ENDPOINT_CONTROL,
+            USB_ENDPOINT_ISOCHRONOUS,
+            USB_ENDPOINT_BULK,
+            USB_ENDPOINT_INTERRUPT
+        } endpoint_type;
+
+        typedef enum usb_subclass
```

Comment:
For review - These name are misleading as these aer not actual USB sub_class featuers

Reply:
I will remove subclass from the API, it will be handled on higher levels (UVC, HID ...)